actors/runtime/src/main/java/cloud/orbit/actors/runtime/ActorEntry.java

```diff
@@ -107,7 +108,7 @@ public T getObject()
                 return Task.fromValue(null);
             }
         }
-        final Object newInstance = runtime.getFirstExtension(LifetimeExtension.class).newInstance(concreteClass);
+        final Object newInstance = runtime.getFirstExtension(ActorConstructionExtension.class).newInstance(concreteClass);
```

Comment:
No, I was actually thinking about that. But it doesn't change the existing behavior so it should be a separate PR.